x-pack/plugins/security_solution/public/overview/routes.tsx

```diff
@@ -17,46 +17,48 @@ import {
 } from '../../common/constants';
 import type { SecuritySubPluginRoutes } from '../app/types';
 
-import { LandingPage } from './pages/landing';
 import { StatefulOverview } from './pages/overview';
 import { DataQuality } from './pages/data_quality';
 import { DetectionResponse } from './pages/detection_response';
 import { PluginTemplateWrapper } from '../common/components/plugin_template_wrapper';
 import { EntityAnalyticsPage } from './pages/entity_analytics';
 import { SecurityRoutePageWrapper } from '../common/components/security_route_page_wrapper';
+import { LandingPage } from './pages/landing';
 
 const OverviewRoutes = () => (
-  <PluginTemplateWrapper>
+  <PluginTemplateWrapper paddingSize="none" restrictWidth={false}>
     <TrackApplicationView viewId={SecurityPageName.overview}>
       <StatefulOverview />
     </TrackApplicationView>
   </PluginTemplateWrapper>
 );
 
 const DetectionResponseRoutes = () => (
-  <PluginTemplateWrapper>
+  <PluginTemplateWrapper paddingSize="none" restrictWidth={false}>
```

Comment:
Why are we adding `paddingSize="none" restrictWidth={false}` to all those _PluginTemplateWrapper_? Only _LandingRoutes_ should need this, only because serverless needs less padding, right?